### PR TITLE
Fix AsanPardakht Config & Work in PHP 7.4

### PR DESCRIPTION
**Pull request by rayansys**

### CODE REVIEW

File: config/payment.php [hunk 1]
```diff
@@ -24,10 +24,10 @@
     */
     'drivers' => [
         'asanpardakht' => [
-            'apiPurchaseUrl' => 'https://services.asanpardakht.net/paygate/merchantservices.asmx?wsdl',
-            'apiPaymentUrl' => 'https://asan.shaparak.ir',
-            'apiVerificationUrl' => 'https://services.asanpardakht.net/paygate/merchantservices.asmx?wsdl',
-            'apiUtilsUrl' => 'https://services.asanpardakht.net/paygate/internalutils.asmx?wsdl',
+            'apiPurchaseUrl' => 'https://ipgsoap.asanpardakht.ir/paygate/merchantservices.asmx?wsdl',
```
Comment (khanzadimahdi) [inline config/payment.php:27]: why you have changed this config file? by changing configs the gateway will work correctly?

Comment (rayansys) [reply config/payment.php:27]: yes

Comment (khanzadimahdi) [reply config/payment.php:27]: could you fix your code according to PSR2? 

see below details

https://github.styleci.io/analyses/lKDgVO?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Comment (khanzadimahdi) [reply config/payment.php:27]: please fix PSR2 in your commits. (the Styleci bot must be passed)

**Comment by khanzadimahdi** [comment]
thanks for your contribution. cool :100:

**Comment by khanzadimahdi** [comment]
version 1.2 released.

https://github.com/shetabit/multipay/releases/tag/v1.2.0

thanks alot.

**Pull request merged**